### PR TITLE
Add PrefixSpan perf tests

### PR DESCRIPTION
**Pull request by feynmanliang**
@mengxr 

Please review now but wait until  #70 merges before merging as merge conflicts will be introduced.

### CODE REVIEW

**Comment by feynmanliang** [comment]
@mengxr Please review

File: config/config.py.template [hunk 1]
```diff
@@ -675,6 +675,19 @@ if MLLIB_SPARK_VERSION >= 1.3:
     MLLIB_TESTS += [("fp-growth", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
         MLLIB_JAVA_OPTS, [ConstantOption("fp-growth")] + MLLIB_FP_GROWTH_TEST_OPTS)]
 
+MLLIB_PREFIX_SPAN_TEST_OPTS = MLLIB_FPM_TEST_OPTS + \
+                            [OptionSet("num-sequences", [5000000], can_scale=True),
```
Comment (mengxr) [inline config/config.py.template]: What is the expected running time? We try to control the running time to be not too short (couple seconds) and not too long (more than 1 minute). If we don't know yet, we can leave a TODO.

**Comment by mengxr** [comment]
LGTM. Made one comment, which we can address after running perf tests.

**Comment by feynmanliang** [comment]
@mengxr Added TODO for comment, will need someone with write access to merge.

**Comment by jkbradley** [comment]
Merging this

**Pull request merged**